commons/ava_testsuite/fully_connected_test/fully_connected_test_.go

```diff
@@ -76,10 +77,10 @@ func (test StakingNetworkFullyConnectedTest) Run(network networks.Network, conte
 
 func (test StakingNetworkFullyConnectedTest) GetNetworkLoader() (networks.NetworkLoader, error) {
 	serviceConfigs := map[networks.ConfigurationID]ava_networks.TestGeckoNetworkServiceConfig{
-		normalNodeConfigId: *ava_networks.NewTestGeckoNetworkServiceConfig(true, ava_services.LOG_LEVEL_DEBUG, test.ImageName, 2, 2),
+		normalNodeConfigId: *ava_networks.NewTestGeckoNetworkServiceConfig(true, ava_services.LOG_LEVEL_DEBUG, test.ImageName, 2, 2, nil),
```

Comment:
Just verifying - passing in `nil`, rather than empty map, won't cause runtime issues here?

Reply:
Yup, ranging over nil will not cause a runtime issue

Reply:
Going to update this to an empty map anyways